benchmarks/synchrobench/src/main/java/com/yahoo/oak/synchrobench/contention/benchmark/OpCounter.java

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright 2020, Verizon Media.
+ * Licensed under the terms of the Apache 2.0 license.
+ * Please see LICENSE file in the project root for terms.
+ */
+
+package com.yahoo.oak.synchrobench.contention.benchmark;
+
+import com.yahoo.oak.synchrobench.contention.abstractions.BenchOp;
+import org.apache.datasketches.kll.KllFloatsSketch;
+
+import java.util.Map;
+import java.util.stream.LongStream;
+import java.util.stream.Stream;
```

Comment:
Again continue general discussion:

This is again very big change to the benchmarks that also serve as our basic performance regression test. We need to verify that our "old" regression still performs the same with the same environment. Plus, I am up to adding 2-3 new regression benchmarks with the new abilities that gives this significant change.